### ISSUE TITLE
Implement or filter

### ISSUE THREAD

**Issue by dreinon** [supabase-community/postgrest-py#137]
Since we have a different way to use the `not` operator than postgrest-js does, an idea for the `or` filter might be:

```py
 supabase_client.table("table")
        .select("*")
        .or()
        .eq("col1", "value1a")
        .eq("col1", "value1b")
        .end_or
        .eq("col2", "value2")
        .execute()
```

The `end_or` can be a property, but `or` has to be a method since the user should be able pass a `foreign_table` as an argument.

**Comment by benrapport** [comment]
This would also be extremely helpful!

**Comment by levity** [comment]
Any plans to implement this soon?

**Comment by silentworks** [comment]
The work on this can be seen in this PR https://github.com/supabase-community/postgrest-py/pull/355

**Comment by silentworks** [comment]
This has now been merged in and available from version 0.14.0 of the Postgrest Python library.
